src/lib/razermouse_driver.c

```diff
@@ -1611,8 +1643,10 @@ ssize_t razer_attr_write_right_mode_reactive(IOUSBDeviceInterface **usb_dev, con
 
 ushort razer_attr_read_dpi(IOUSBDeviceInterface **usb_dev)
 {
-    struct razer_report report, response_report;
+    struct razer_report report;
+    struct razer_report response_report = {0};
```

Comment:
Any reason for this change?

Reply:
Not really just looked like it conformed to the rest of the code. but I'll revert this as well.